generator/intermediate_data.cpp

```diff
@@ -145,61 +133,131 @@ class RawMemPointStorageReader : public PointStorageReaderInterface
   std::vector<LatLon> m_data;
 };
 
-// RawMemPointStorageWriter ------------------------------------------------------------------------
 class RawMemPointStorageWriter : public PointStorageWriterBase
 {
+  // 16G buffer size.
+  static constexpr size_t kBufferSize = 1000000000;
```

Comment:
It'll be used on every generator start, right?
Then 16G will make it unusable on many devs' personal computers?

Reply:
This is used for NODE_STORAGE=mem only (>= 128G RAM, AFAIR).